lua/presenting/init.lua

```diff
@@ -50,15 +50,16 @@ Presenting.config = {
     adoc = "^==+ ",
     asciidoctor = "^==+ ",
   },
+  keep_separator = true,
```

Comment:
Please also document this behavior, so that it appears in the helptext. Maybe even mention your use case.